src/scripts/build-repo-data.js

```diff
@@ -28,7 +28,8 @@ const repoNames = [
     { owner: "ExpediaGroup", name: "pitchfork" },
```

Comment:
i suggest we replace this one with pitchfork.
thoughts from the oss folks?

Reply:
Sure.

Reply:
@worldtiki i did the requested changes

Reply:
Previewed locally, looks good :)

![Screen Shot 2021-07-14 at 11 46 24 AM](https://user-images.githubusercontent.com/44259532/125601548-6b566808-5942-45eb-a792-13fa624eac75.png)

Reply:
@fborriello would it make sense, for the future, to have a logo with "Expedia Group" instead of "Hotels.com"?

Reply:
sure! we even need to change the colour, we should contact the guy who did the image the first time

Reply:
@ffeltrinelli I noticed that the build workflow has failed...

Reply:
Yes unfortunately, it's due to recent changes to the org-level permissions, I'll have a look asap! :-)

Reply:
Ok fixed, you are now live! :-)

Reply:
Thank you